resources/language/resource.language.nl_nl/strings.po

```diff
@@ -95,19 +96,19 @@ msgstr "Het commando [B]'{command}'[/B] moet op het systeem aanwezig zijn om [B]
 
 msgctxt "#30022"
 msgid "Downloading the ChromeOS recovery image..."
-msgstr "Recovery-image van ChromeOS downloaden..."
+msgstr "De recovery-image van Chrome OS wordt gedownload..."
 
 msgctxt "#30023"
 msgid "Download completed!"
 msgstr "Klaar met downloaden!"
 
 msgctxt "#30024"
 msgid "The installation now needs to unzip, mount and extract [B]Widevine CDM[/B] from the recovery image.[CR][CR]This process may take up to ten minutes to complete."
-msgstr "Voor de installatie moet het recovery-image worden gedecomprimeerd en gemount en moet [B]Widevine CDM[/B] worden uitgepakt.[CR][CR]Dit proces kan wel tien minuten duren."
+msgstr "Voor de installatie moet de recovery-image worden gemount, worden gedecomprimeerd en moet [B]Widevine CDM[/B] worden uitgepakt.[CR][CR]Dit proces kan wel tien minuten duren."
```

Comment:
Volgens mij is het "het image" en niet "de image". Maar dat is puur op gevoelsbasis :-)

Reply:
Ik heb het met Nederlands docent besproken: het is "Het image" als het over het imago gaat. Aangezien het hier om de Engelse variant van afbeelding gaat is het "de recovery image"

Reply:
Je krijgt toch niet het lidwoord van de vertaling in het Nederlands? Als ik kies voor de vertaling van "het beeld" zou het dan ook onzijdig worden. Dat lijkt me zeer dubbelzinnig en dus twijfelachtig.

Of het nu recovery-image is of herstel-image, dat heeft ook geen invloed op het geslacht. Van Dale zegt ook dat het onzijdig is: https://www.vandale.nl/gratis-woordenboek/nederlands/betekenis/image

Reply:
Van Dale doelt op het imago van mensen (_indruk die de buitenwereld heeft van iem. of iets; = imago_). Het gaat hier om het leen-woord _Image_ vanuit het Engels: https://taaladvies.net/taal/advies/tekst/40/woordgeslacht_van_engelse_leenwoorden_algemeen/

Reply:
En de/het image blijkt dus allebei juist volgens de taalunie.
https://onzetaal.nl/taaladvies/engelse-woorden-de-het-weblog/

Reply:
de recovery image is voor mij oké. Microsoft en Dell gebruiken dat die term ook op hun Nederlandstalige support-pagina's.
_herstelinstallatiekopie_ is ook een woord http://www.mijnwoordenboek.nl/vertaal/EN/NL/recovery%20image